### ISSUE TITLE
Insights: refactor generateInsight to either use summaries or not

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#10039]
This PR https://github.com/ParabolInc/parabol/pull/10017 created the `generateInsight` mutation that lets us test whether it's best to send all of the meeting data (for Parabol teams) to ChatGPT or summaries of meetings. 

Once we've decided on an approach, refactor the mutation to either `useSummaries` or not.

### AC

- [ ] Remove `getTopics` or `getSummaries` from [L22](https://github.com/ParabolInc/parabol/blob/5c191ef5776d3ff1f39eda67a7e01270d8486870/packages/server/graphql/public/mutations/generateInsight.ts)

**Comment by jordanh** [comment]
@nickoferrall what's your feeling on this? For me, I think the summaries lose too much detail.

**Comment by nickoferrall** [comment]
@jordanh I agree, I think we should actually keep both. Not using the summaries results in the best quality output, but if users want to create insights on the fly in the near future, a summary of summaries works well. 

I suggest we close this issue and keep the mutation as it is.

**Comment by jordanh** [comment]
I concur. I will close the issue now.
